utils_cv/tracking/model.py

```diff
@@ -163,18 +120,23 @@ def __init__(
         self.opt.device = torch_device()
 
         self.dataset = dataset
-        self.model = self.init_model()
         self.model_path = model_path
+        self.model = self.init_model()
 
     def init_model(self) -> nn.Module:
         """
         Download and initialize the baseline FairMOT model.
         """
-        model_dir = osp.join(self.opt.root_dir, "models")
-        baseline_path = osp.join(model_dir, "all_dla34.pth")
-        #         os.makedirs(model_dir, exist_ok=True)
-        #         _download_baseline(BASELINE_URL, baseline_path)
-        self.opt.load_model = baseline_path
+        if osp.isfile(self.model_path):
```

Comment:
Could rename function to "_init_model()" to indicate it's only used internally, ie the user will never call it.